### ISSUE TITLE
.env.local file doesn't appear in file search

### ISSUE THREAD

**Issue by mifopen** [zed-industries/community#1538]
### Check for existing issues

- [X] Completed

### Describe the bug / provide steps to reproduce it

1. Create file with name `.env.local` in the open directory
2. Open file search pallete
3. Search for `env` or `local`

### Environment

Zed: v0.85.4 (stable)
OS: macOS 13.3.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, attach your `~/Library/Logs/Zed/Zed.log` file to this issue.
If you only need the most recent lines, you can run the `zed: open log` command palette action to see the last 1000.

_No response_

**Comment by JosephTLyons** [comment]
Hey @mifopen, by chance, do you have the `.env.local` file `.gitginore`d?

**Comment by mifopen** [comment]
@JosephTLyons yes, 100%

**Comment by mifopen** [comment]
And I see where it could come from. But I've been using JetBrains IDEs for a decade and believe that "having smth in .gitignore" doesn't equal "excluded from the global project search". It's a separate setting (you can mark directories/files as "excluded") in JB IDEs and it feels natural.

**Comment by JosephTLyons** [comment]
I think at the moment, that behavior is baked into zed on purpose, to prevent things like node_modules (and similar setups in other languages) from absolutely polluting the file finder results.  That being said, we should probably consider some sort of setting here:

- https://github.com/zed-industries/community/issues/70

**Comment by mifopen** [comment]
Totally agree

**Comment by TwanLuttik** [comment]
I agree as well but maybe add a option in the setting to override it.
